test/util_test.dart

```diff
@@ -591,8 +591,8 @@ void overReactExample() {}''';
 
       test('returns empty list when input has no descendants', () {
         final node = parseAndGetSingle('''
-          UiFactory<FooProps> Foo = castUiFactory(_\$Foo); // ignore: undefined_identifier
-        ''').variables.variables.first.root;
+          var foo = bar;
+        ''').variables.variables.first.initializer!;
```

Comment:
I swapped this with a simpler snippet so it was easier to reason about, and now node is `bar` by using `.initializer`.